src/algebra/algebra/spectrum.lean

```diff
@@ -37,6 +37,8 @@ This theory will serve as the foundation for spectral theory in Banach algebras.
 * `σ a` : `spectrum R a` of `a : A`
 -/
 
+open set
+
```

Comment:
This seems pretty pointless given you don't edit the rest of the file at all!

Reply:
This is necessary, see the PR description. I'm putting a bunch of lemmas in the `set` namespace, as they should have been. Do you want that in a separate PR?

Reply:
Can you list those lemmas in the description?

Reply:
Split off to #14281.